docs/api-reference/indexing/create_index.mdx

```diff
@@ -0,0 +1,316 @@
+---
+title: Create an Index
+---
+
+The following code block creates an index called `search_idx` over multiple columns of the `mock_items` table.
+```sql
+CALL paradedb.create_bm25(
+  index_name => 'search_idx',
+  table_name => 'mock_items',
+  key_field => 'id',
+  text_fields => paradedb.field('description') || paradedb.field('category'),
+  numeric_fields => paradedb.field('rating'),
+  boolean_fields => paradedb.field('in_stock'),
+  datetime_fields => paradedb.field('created_at'),
+  json_fields => paradedb.field('metadata')
+);
+```
+
+<ParamField body="index_name" required>
+  The name of the index. The index name can be anything, as long as doesn't
+  conflict with an existing index or schema. A new schema with associated query
+  functions will be created with this name.
+</ParamField>
+<ParamField body="table_name" required>
+  The name of the table being indexed.
+</ParamField>
+<ParamField body="key_field" required>
+  The name of a column in the table that represents a unique identifier for each
+  record. Usually, this is the same column that is the primary key of the table.
+  In version 0.7.5 and earlier, only integer IDs were supported. From version
```

Comment:
I think we should remove this disclaimer. We're 2 minor versions above 0.7 and any new user won't be facing this. As for old users that may be on 0.7+ still so much has changed that they need to update anyway and will see that it is supported.